types/gauge/index.d.ts

```diff
@@ -0,0 +1,129 @@
+// Type definitions for gauge 2.7
+// Project: https://github.com/iarna/gauge
+// Definitions by: Brian J Brennan <https://github.com/brianloveswords>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace Gauge {
+    interface Stream { }
```

Comment:
TypeScript uses a structural type system, so an empty interface is equivalent to the `{}` types, which accepts any value other than `null` or `undefined`.